### PR TITLE
Fix several typos and missing spaces in Op docs.

### PR DESCRIPTION
**Pull request by kit1980**

### CODE REVIEW

**Comment by kit1980** [comment]
Any interest in fixing these?

File: onnx/defs/tensor/defs.cc [hunk 1]
```diff
@@ -2312,7 +2312,7 @@ ONNX_OPERATOR_SET_SCHEMA(
             1,
             "indices",
             "A 1-D INT64 tensor "
-            "containing indices of 'Y' elements' first occurance in 'X'. "
+            "containing indices of 'Y' elements' first occurence in 'X'. "
```
Comment (gramalingam) [inline onnx/defs/tensor/defs.cc]: occurrence

File: onnx/defs/tensor/defs.cc [hunk 2]
```diff
@@ -2210,7 +2210,7 @@ Otherwise the input tensor is flattened and unique values of the flattened tenso
 
 This operator returns the unique values or sliced unique subtensors of the input tensor and three optional outputs. 
 The first output tensor 'Y' contains all unique values or subtensors of the input. 
-The second optional output tensor 'indices' contains indices of 'Y' elements' first occurance in 'X'.. 
+The second optional output tensor 'indices' contains indices of 'Y' elements' first occurence in 'X'.
```
Comment (gramalingam) [inline onnx/defs/tensor/defs.cc]: occurrence

File: onnx/defs/gen_doc.py [hunk 3]
```diff
@@ -214,6 +214,14 @@ def support_level_str(level):  # type: (OpSchema.SupportType) -> Text
         "<sub>experimental</sub> " if level == OpSchema.SupportType.EXPERIMENTAL else ""
 
 
+class Schema(object):
```
Comment (gramalingam) [inline onnx/defs/gen_doc.py]: Can you add a brief note why the changes in this file are needed? I vaguely remember seeing a thread (you may have written) long back about some motivating problem. It would be good to document it. Thanks!

Comment (kit1980) [reply onnx/defs/gen_doc.py]: Sorry, I didn't intent this tp be a part of this PR.

**Comment by linkerzhang** [comment]
@kit1980 do you want to resolve the comments of removing the unnecessary codes in the PR?
